### DISCUSSION TITLE
HtmlRenderer

### DISCUSSION THREAD

**Discussion by trashhacker** [AvaloniaUI/Avalonia#17185] [Q&A]
U dont have a discussion on the repository for the HtmlRenderer, so i ask my question here. 
is the renderer able to resolve the fallowing. if not how could i write a workaround?

      @using Syncfusion.EJ2
      <html>
          <head>
              <link rel="stylesheet" href="https://cdn.syncfusion.com/ej2/27.1.48/fluent.css" />
              <script src="https://cdn.syncfusion.com/ej2/27.1.48/dist/ej2.min.js"></script>
          </head>
          <body>
              <div>
                  <h3>SimpleExample</h3>
              </div>
              <div>
                  <ejs-pdfviewer id="pdfviewer" style="height:600px" documentPath="https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf"/>
              </div>
              <ejs-scripts></ejs-scripts>
          </body>
      </html>
      
      @code {
      }

ah yea im processing it with blazor so the output is this

    <html>
        <head>
            <link rel="stylesheet" href="https://cdn.syncfusion.com/ej2/27.1.48/fluent.css">
            <script src="https://cdn.syncfusion.com/ej2/27.1.48/dist/ej2.min.js"></script>
        </head>
        <body>
            <div>
                <h3>SimpleExample</h3>
            </div>
            <div>
                <ejs-pdfviewer id="pdfviewer" style="height:600px" documentPath="https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf">
               </ejs-pdfviewer>
           </div>
              <ejs-scripts></ejs-scripts>
        </body>
    </html>

do u think this would work https://github.com/MicroSugarDeveloperOrg/Avalonia.WebView

**Comment by maxkatz6** [comment]
Avalonia.HtmlRenderer doesn't support dynamic content. If you have a static HTML, then it should work.

Full-featured webviews should work either way, yes.
